data/local/src/main/java/com/ku_stacks/ku_ring/local/entity/KuringBotMessageEntity.kt

```diff
@@ -0,0 +1,12 @@
+package com.ku_stacks.ku_ring.local.entity
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+data class KuringBotMessageEntity(
+    @PrimaryKey(autoGenerate = true) val id: Int,
```

Comment:
```suggestion
    @PrimaryKey(autoGenerate = true) val id: Long,
```

로 바꾸는건 어떤가요? 메시지여서 생각보다 금방 쌓일수도...?

Reply:
월 2회 제한이 걸려있기도 하고, 나중에 좀 쌓인다 싶을 때 바꿔도 될 거 같아요. SQL 변경 없이 Entity만 바꾸면 되는 거라.